src/libraries/System.Net.Quic/tests/FunctionalTests/QuicConnectionTests.cs

```diff
@@ -22,10 +23,15 @@ public sealed class QuicConnectionTests : QuicTestBase
 
         public QuicConnectionTests(ITestOutputHelper output) : base(output) { }
 
-        [Theory]
+        [ConditionalTheory]
         [MemberData(nameof(LocalAddresses))]
         public async Task TestConnect(IPAddress address)
         {
+            if (address.AddressFamily == AddressFamily.InterNetworkV6 && !IsIPv6Available)
+            {
+                throw new SkipTestException("IPv6 is not available on this platform");
```

Comment:
This is ok... but could LocalAddresses instead only yield IPv6 addresses if IsIPv6Available is true?

Reply:
Moved the check to LocalAddresses